book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Please make sure that Getters and Setters for `@rentals` should be present.

Reply:
- [ ] As I mentioned before, Please add Getters and Setters `attr_accessor` for `@rentals`, not Getters only `attr_reader` as you did.